client/valor/schemas/symbolic/collections.py

```diff
@@ -135,8 +145,14 @@ def decode_value(cls, value: dict):
 
     def encode_value(self):
         """Encode object to JSON compatible dictionary."""
+        # TODO: make this less hacky
         return {
-            k: v.encode_value() for k, v in self._get_dynamic_values().items()
+            k: (
+                v.encode_value()
+                if v is not None and hasattr(v, "encode_value")
+                else v
```

Comment:
what condition causes this `v = None`? A `StaticCollection` should convert build-in types to symbolic types at initialization

Reply:
symbolic attributes can contain the value `None` but are not themselves nullable

Reply:
yea i think you're right, just dropped that check